cpu/lm4f120/periph/uart.c

```diff
@@ -28,6 +28,15 @@
 #include "periph_conf.h"
 
 /**
+ * @brief Unified interrupt handler for all UART devices
+ *
+ * @param uartnum       the number of the UART that triggered the ISR
+ * @param ulBase        the UART device that triggered the ISR
+ * @param ulRxInt       the RX Interrupt Mask of the UART Device that triggered the ISR
+ */
+static inline void irq_handler(uart_t uartnum, unsigned long ulBase, unsigned long ulRxInt);
```

Comment:
@rakendrathapa could you remove the camel case here like pointed out by @thomaseichinger (: